db/dcrpg/internal/stats.go

```diff
@@ -0,0 +1,71 @@
+// Copyright (c) 2019, The Decred developers
+// See LICENSE for details.
+
+package internal
+
+// The stats table holds additional data beyond basic block data. row are unique
+// on height, so this table does not retain information related to orphaned
+// blocks.
+const (
+	CreateStatsTable = `
+		CREATE TABLE IF NOT EXISTS stats (
+    blocks_id INT4 REFERENCES blocks(id) ON DELETE CASCADE,
+    height INT4,
+    pool_size INT8,
+    pool_val INT8
+  );`
```

Comment:
I don't know why this happens with my editor, but I've seen this before and I don't know how to fix it. Only happens in multi-line string literals, and even then only sometimes.